### PR TITLE
Fixed dash example

### LINKED ISSUE #8188: Example app "dashboard" unsuccessful on wide screen
All wide plots get too tall and everything gets cut off

screenshot from 2018-08-25 17-46-36

Simplest first step would be to set a max-width on the dashboard css.

### PR DESCRIPTION
**Pull request by PabloRuizCuevas**
All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [X] issues: fixes #8188
- [x] tests added / passed
- [ ] release document entry (if new feature or API change)

I took the changes proposed #8188 and added a couple of tweaks so it works better and it is displayed properly on edge too. I also changed the css to a non minified version, as I also changed the code there, feel free to minify it again.

### CODE REVIEW

**Comment by bryevdv** [comment]
@PabloRuizCuevas can you post a few screenshots at various widths? I don't know when I would be able to check this out locally but I would merge w/ some screenshots as documentation here.

**Comment by PabloRuizCuevas** [comment]
image

image

image

File: examples/server/app/dash/main.py [hunk 1]
```diff
@@ -2,10 +2,9 @@
 This example demonstrates the use of Bootstrap templates to compile several types
 of charts in one file.
 
-.. note::
+note:
     This example needs the Pandas package to run.
-
-.. _Auto MPG: https://archive.ics.uci.edu/ml/datasets/auto+mpg
+    _Auto MPG: https://archive.ics.uci.edu/ml/datasets/auto+mpg
```
Comment (bryevdv) [inline examples/server/app/dash/main.py:7]: @PabloRuizCuevas All the changes to the docstring need to be reverted. The removed bits are Sphinx directives and are there intentionally.

Comment (PabloRuizCuevas) [reply examples/server/app/dash/main.py:8]: @bryevdv ups true, now is fixed, thanks for the feedback.

**Comment by PabloRuizCuevas** [comment]
The test that fail are in "multi_choice" and "test_toggle" that I'm not using, I think this is good to go.

**Comment by mattpap** [comment]
> The test that fail are in "multi_choice" and "test_toggle" that I'm not using, I think this is good to go.

Those are "known" failures (see issue #9597).

**Comment by mattpap** [comment]
What happened to `custom.min.css`? If there were no updates, then the original should be restored.

**Comment by PabloRuizCuevas** [comment]
> What happened to `custom.min.css`? If there were no updates, then the original should be restored.

I thought for an example i.e not production code, it made no much sense to have a unreadable css, so I just de-minify it, I included some little differences, so it is not any-more 100% same as it was before, but I can minify this version if it is required.

**Comment by bryevdv** [comment]
So `custom.css` is a customized bootstrap css? I guess ideally we would load a stock (minified) bootstrap and then a much smaller (unminified) `custom.css` with the customzations. What do you think @PabloRuizCuevas ?

**Comment by bryevdv** [comment]
OK I will go ahead and merge this as-is, we can continue to refine in future in future PRs. @PabloRuizCuevas if you have bandwidth to look a little more at this, I do think "stock bootstrap + small custom CSS" would be the best thing to show for users.

**Comment by PabloRuizCuevas** [comment]
@bryevdv Sure thanks I will try to take care of that modification, I just had no time last week, I will add a issue and assign it to myself.

**Pull request merged**